Makefile

```diff
@@ -45,7 +45,14 @@ update-scripts:
 	hack/update-deepcopy.sh
 	hack/update-clientgen.sh
 .PHONY: update-scripts
-update: update-scripts update-codegen-crds update-manifests
+update: update-scripts update-codegen-crds update-manifests update-crds
+
+# FIXME: crd-schame-gen.mk should contain all the necessary bits to generate CRDs. Nevertheless it
+# renders incorrect command (without crd parameter) to controller-gen.
+update-crds: ensure-controller-gen
+	'$(CONTROLLER_GEN)' crd crd:crdVersions="v1" schemapatch:manifests="./bundle/cert-manager-operator/manifests" paths="./apis/operator/v1alpha1" output:dir="./bundle/cert-manager-operator/manifests"
+	'$(CONTROLLER_GEN)' crd crd:crdVersions="v1" schemapatch:manifests="./bundle/cert-manager-operator/manifests" paths="./apis/config/v1alpha1" output:dir="./bundle/cert-manager-operator/manifests"
+.PHONY: update-crds
```

Comment:
Instead of this, add
```
$(call add-crd-gen,operator-alpha,./apis/config/v1alpha1,./bundle/cert-manager-operator/manifests,./bundle/cert-manager-operator/manifests)
```
to the similar line at line 23.

Feel free to update the `CONTROLLER_GEN_VERSION` to `v0.6.0`

You can then update the CRDs with `make update-codegen-crds`

Reply:
This still doesn't work. After the line as you said, I'm getting:

```
$ make update-codegen-crds
Makefile:24: warning: overriding recipe for target 'update-codegen-crds-operator-alpha'
Makefile:23: warning: ignoring old recipe for target 'update-codegen-crds-operator-alpha'
Makefile:24: warning: overriding recipe for target 'verify-codegen-crds-operator-alpha'
Makefile:23: warning: ignoring old recipe for target 'verify-codegen-crds-operator-alpha'
Using existing controller-gen from "_output/tools/bin/controller-gen"
Using existing yq from "_output/tools/bin/yq"
Using existing yaml-patch from "_output/tools/bin/yaml-patch"
'_output/tools/bin/controller-gen' schemapatch:manifests="./bundle/cert-manager-operator/manifests" paths="./apis/config/v1alpha1" output:dir="./bundle/cert-manager-operator/manifests"

$ ll bundle/cert-manager-operator/manifests/
total 0
```

So as you can see - it doesn't generate anything. Changing `CONTROLLER_GEN_VERSION` to `v0.6.0` doesn't fix it.

Reply:
The file for the given type needs to be pre-existent. Is it?

Reply:
Done.